src/app/static-pages/landing-page/sections/intro-section/intro-section.component.html

```diff
@@ -1,19 +1,24 @@
-<div class="content">
-  <div class="text">
-    <h1 class="title">Gemeinsam gegen Falschinformationen</h1>
-    <p class="paragraph">      
-      Bei codetekt kannst du digitale Informationen auf ihre Vertrauenswürdigkeit prüfen lassen. 
-      Reiche einen Fall ein und lasse diesen von der codetekt Community prüfen oder werde selbst Detektiv*in.
-      Beim Lösen der Fälle kannst du deine Recherche-Fähigkeiten verbessern und vom Einsteiger- zum Expertenlevel aufsteigen.
-    </p>
-    <div class="button-group">
-      <a [routerLink]="['/submit']" class="button button--bittersweet"><i class="far fa-typewriter"></i>Fall einreichen</a>
-      <a [routerLink]="['/review']" class="button button--neon-blue"><i class="far fa-clipboard-list-check"></i>Fall bearbeiten</a>
-      <a [routerLink]="['/archive']" class="button button--neon-blue button--inverted">
-        <i class="far fa-cabinet-filing"></i>
-        Gelöste Fälle ansehen
-      </a>
+<div class="top-section standard-content-container-width">
+  <app-breadcrumb [links]="breadcrumbLinks"></app-breadcrumb>
```

Comment:
Ich finde die Breadcrumbs auf der Startsteite eher störend, da man ja sowieso schon auf der "Home"-Seite ist. 
Müsste die Startseite nicht eigentlich nur "Home" als Breadcrumb haben, statt "Home > Überblick"? So wie es hier ist, hätte "Home" gar keine Funktion.

Was hältst du davon, die Breadcrumbs hier nur anzuzeigen, wenn man über '/landingpage' zugreift und über '/' nicht anzuzeigen? Folgerichtig könnten wir dann beim Dashboard nach Login die Breadcrumbs auch weglassen. 
Dann würde ich sie

Reply:
Wenn du über den Footer oder Menülink auf die Startseite kommst bist du immer auf '/landingpage'. Wenn du auf das Logo klickst, landest du auf '/' (schau dir das mal in der URL-Leiste im Browser an). 

Mein Vorschlag war, dass wir die Breadcrumbs nur anzeigen, wenn wir auf '/landingpage' sind - also wenn der User aktiv über Header oder Footer zu dieser Seite navigiert hat. Auf '/' würde ich es nicht zeigen, da es für Leute, die das erste Mal auf codetekt.org landen vielleicht eher irritierend wirkt.

Reply:
Klar, das hatte ich schon verstanden.